stable/chronograf/values.yaml

```diff
@@ -29,10 +34,23 @@ resources:
     
 ## Configure the ingress object to hook into existing infastructure
 ## ref : http://kubernetes.io/docs/user-guide/ingress/
+## OPTIONALLY you can set .Values.ingress.secretName to set which secret to use    
 ## 
 ingress:
   enabled: false
   tls: false
   hostname: chronograf.foobar.com
   annotations:
-    kubernetes.io/ingress.class: "nginx"
\ No newline at end of file
+    kubernetes.io/ingress.class: "nginx"
+    secretName: my-tls-cert
+  # kubernetes.io/acme-tls: "true"
+
+## OAuth Settings for Github OAuth
+##
+oauth:
+  enabled: false
+  github:
+    client_id: f612alskdfc91cad5ad
```

Comment:
I would make that very clear by commenting this section out as they are not useful and ensuring that the value is something that denotes that it needs to be changed (ie `CHANGE_ME`)